pwd-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: "3"
+
+volumes:
+  starter_kit_app: {}
+  mysql_config: {}
+
+services:
+  source:
+    image: containerize/git
+    environment:
+      - REPO_LOCAL=/data/git/repos/yii2-starter-kit
+      - REPO_BRANCH=master
+      - REPO_REMOTE=https://github.com/yii2-starter-kit/yii2-starter-kit.git
+    tty: true
+    volumes:
+      - starter_kit_app:/data/git/repos/yii2-starter-kit
+      - mysql_config:/data/git/repos/yii2-starter-kit/docker/mysql
+
+  app:
+    image: alfredleo/starter_kit_app:secondtry
```

Comment:
This section is the only thing that troubles me.

Is it using an image that you're building with an external build system?